src/routes/Auth.vue

```diff
@@ -34,18 +40,23 @@ export default {
     },
     mounted() {
         const uiConfig = {
-            signInSuccessUrl: '/game',
+            signInSuccessUrl: '/',
             signInOptions: [
                 firebase.auth.GoogleAuthProvider.PROVIDER_ID,
                 firebase.auth.EmailAuthProvider.PROVIDER_ID
                 ]
         };
-        let ui = new firebaseui.auth.AuthUI(firebase.auth());
+        let ui = firebaseui.auth.AuthUI.getInstance();
+        if (!ui) {
+            ui = new firebaseui.auth.AuthUI(firebase.auth());
+        }
```

Comment:
You can use the `??` operator in ES2017+.

```js
let ui = firebaseui.auth.AuthUI.getInstance() ?? new firebaseui.auth.AuthUI(firebase.auth());
```

Reply:
Nice, marked to change later